examples_tests/35.TAA/main.cpp

```diff
@@ -311,6 +311,8 @@ int main()
         else
         {
             prevVP = currVP;
+            // TODO how to force driver to update view and projection matrices without smgr->drawAll()?
+            // without this fix, here i'm getting previous frame's VP to currVP (and so prevVP is 2 frames in the past)
```

Comment:
get the transforms from the camera, and not the driver... also refactor into an extension like I told you on discord, then there is absolutely no problem with history tracking